__tests__/app.test.js

```diff
@@ -18,6 +18,18 @@ afterAll(() => {
   db.end();
 });
 
+describe.only("GET /api", () => {
+  it("200: responds with instruction on how to use endpoints", async () => {
+    const response = await request(app).get("/api");
+    const expectedOutput = await fs.readFile("./endpoints.json", "utf-8");
+    expect(response.status).toBe(200);
+    expect(response.body.endpoints).toHaveProperty("GET /api");
+    expect(response.body.endpoints).toHaveProperty("GET /api/topics");
+    expect(response.body.endpoints).toHaveProperty("GET /api/articles");
+    expect(response.body.endpoints).toEqual(JSON.parse(expectedOutput));
```

Comment:
Nice one here with just reading the file and expecting the response to equal the contents of the file, we don't also need to have the previously 3 assertions as we are just duplicating our testing here. We already know the response matches the entire contents.